style/road-colors-generated.mss

```diff
@@ -4,23 +4,23 @@
 /*                                                                    */
 /* scripts/generate_road_colours.py > style/road-colors-generated.mss */
 /*                                                                    */
-@motorway-low-zoom-casing: #c24e6b;
-@trunk-low-zoom-casing: #cf6649;
-@primary-low-zoom-casing: #c38a27;
-@secondary-low-zoom-casing: #9eae23;
 @motorway-casing: #dc2a67;
 @trunk-casing: #c84e2f;
 @primary-casing: #a06b00;
 @secondary-casing: #707d05;
-@motorway-shield: #620728;
-@trunk-shield: #5d1b0b;
-@primary-shield: #4c2e00;
-@secondary-shield: #323b00;
-@motorway-low-zoom: #e66e89;
-@trunk-low-zoom: #f5977a;
-@primary-low-zoom: #f3c380;
-@secondary-low-zoom: #e8eda0;
 @motorway-fill: #e892a2;
 @trunk-fill: #f9b29c;
 @primary-fill: #fcd6a4;
 @secondary-fill: #f7fabf;
+@motorway-low-zoom: #e66e89;
+@trunk-low-zoom: #f6967a;
+@primary-low-zoom: #f4c37d;
+@secondary-low-zoom: #e7ed9d;
+@motorway-low-zoom-casing: #c24e6b;
+@trunk-low-zoom-casing: #d1684a;
+@primary-low-zoom-casing: #c78d2b;
+@secondary-low-zoom-casing: #a4b329;
+@motorway-shield: #620728;
+@trunk-shield: #5f1c0c;
+@primary-shield: #503000;
+@secondary-shield: #364000;
```

Comment:
The new `@primary-shield` color `#503000` is only 1 lighter and 1 higher chroma by Lch, compared to the old `#4c2e00` - this is barely perceptible, like the changes above. 

But the `@secondary-shield` color `#364000` (Lch 25,36,111) was changed from `#323b00` (Lch 23,34,111) so that's 2 ligheter and 2 higher chroma, which might be visible, so it would be good to test the new rendering.